internal/common/environment.go

```diff
@@ -0,0 +1,21 @@
+package common
+
+import "os"
+
+const (
+	templateValidatorImage = "VALIDATOR_IMAGE"
+)
+
+func envOrDefault(envName string, defVal string) string {
+	val := os.Getenv(envName)
+	if val == "" {
+		return defVal
+	}
+	return val
+}
+
+func GetTemplateValidatorImage() string {
+	const defaultVersion = "v0.6.6"
```

Comment:
there is a new version v0.7.0 https://github.com/kubevirt/kubevirt-template-validator/releases/tag/v0.7.0

Reply:
Updated